### PR TITLE
Added a Windows setup section to the ReadMe

### PR DESCRIPTION
**Pull request by Plutoren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,42 @@ Shuttle Tracker needs configuration to run properly. The preferred method during
...
+6. Switch to the Shuttle Tracker directory (`$GOPATH/src/github.com/wtg/shuttletracker`)
+7. Run `govendor sync`
+![Steps 4-6](https://user-images.githubusercontent.com/6248819/47017579-9db76280-d120-11e8-8de5-ab5cbe11e072.png)
+8. Download and run the installer PostgreSQL [from here](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads). Select the latest version. When prompted to set a password, make it something simple, as you will be using this later, for example `shuttle`.  Use default options for everything else. Write down your password.
+![Step 8](https://user-images.githubusercontent.com/6248819/47017593-ac057e80-d120-11e8-8637-18307ebeaf7e.png)
+![Step 8b](https://user-images.githubusercontent.com/6248819/47017613-b6c01380-d120-11e8-85ee-35442f6ea737.png)
+![Step 8c](https://user-images.githubusercontent.com/6248819/47017633-c2abd580-d120-11e8-95ab-50088fb38c40.png)
+9. When complete, open pgAdmin from your search bar. In the Object Browser, open Servers, then open your PostgreSQL server. You will need to enter your password from step 8. Once this is done, right click on Databases and select New Database. Name it `shuttletracker` and hit Ok.
```
Comment (garoller) [inline README.md]: May want to add that if a user doesn't see pgAdmin in their search after postgres is installed that they should restart their computer.

File: README.md [hunk 2]
```diff
@@ -34,3 +34,42 @@ Shuttle Tracker needs configuration to run properly. The preferred method during
 ### Environment variables
 
 Most keys can be overridden with environment variables. The variables names usually take the format `SECTION_KEY`. For example, overriding database's Mongo URL could be done with a variable named `POSTGRES_URL`.
+
+
+## Setting up (Windows)
+
```
Comment (garoller) [inline README.md]: I installed Go (unsuccessfully) a few months ago, but I didn't uninstall the older version before following these steps, which caused some issues. For example, my `GOPATH` was set correctly, but the folder `C:/Users/rolleg/go` did not exist. I could also run `go get ...`, but it wasn't working or generating any sort of error message.

I would add a step to uninstall any unsuccessful Go install before starting and provide [this link](https://golang.org/doc/install#uninstall).

**Comment by Plutoren** [comment]
Let me know if I missed anything or if any further changes need to be made.

**Comment by garoller** [comment]
There have been some updates to the README on `wtg:master`. If you could resolve those merge conflicts locally and then push those changes, that would be great!

**Comment by Plutoren** [comment]
I realize now that the actual main setup has changed I will update the windows section to reflect these changes soon

**Pull request merged**